### ISSUE TITLE
A critical error has occurred! A problem occurred during communication with Opencast.  Details:  Status 404: Event '913c5b23-fc24-4958-b9ed-c6b9d8114fa4' not Found

### ISSUE THREAD

**Issue by oas777** [elan-ev/tobira#1252]
https://tobira-test-oc.ethz.ch/editor-ui/index.html?id=913c5b23-fc24-4958-b9ed-c6b9d8114fa4&callbackUrl=https%3A%2F%2Ftobira.opencast.org%2F%7Emanage%2Fvideos%2FDdXFrrcIKRU&callbackSystem=Tobira+Test+Deployment
bzw.
https://tobira.opencast.org/!v/DdXFrrcIKRU beim Öffnen des Editors

**Comment by LukasKalbertodt** [comment]
I can reproduce that. Interestingly, when I log into Opencast as admin, and then reopen the editor, I get a different error: "Status 400: No internal publication. Event ID: 913c5b23-fc24-4958-b9ed-c6b9d8114fa4". This seems to be the same error you reported on your internal issue tracker.

Maybe summoning @Arnei to take a look? If you need more information on how to test this or access to the Opencast, just let me know.

**Comment by LukasKalbertodt** [comment]
Ok I just talked to Piri and learned a bit about ETH workflows and how they used to remove internal publications and stuff. With that background, I would just close this issue. In general: all old videos on the test system are in a potentially dubious state. We imported them years ago and who knows what's broken about them. I'd say: if you find an editor problem with a video you uploaded yourself, then that's certainly a bug worth investigating. But if single older videos randomly fail (on our public Tobira test server), that's probably no cause for concern.
